### ISSUE TITLE
Traceback, AttributeError

### ISSUE THREAD

**Issue by bogdanstoian1988** [DIGITALCRIMINAL/UltimaScraper#1455]
```
Scrape: 0 = All | 1 = Images | 2 = Videos | 3 = Audios | 4 = Texts
0
Type: Profile
Type: Stories
Scrape Attempt: 1/100
Processing Scraped Stories
Traceback (most recent call last):
  File "/mnt/d/OnlyFans-master/start_ofd.py", line 60, in <module>
    loop.run_until_complete(main())
  File "/usr/local/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/mnt/d/OnlyFans-master/start_ofd.py", line 44, in main
    api = await main_datascraper.start_datascraper(
  File "/mnt/d/OnlyFans-master/datascraper/main_datascraper.py", line 93, in start_datascraper
    await main_helper.process_names(
  File "/mnt/d/OnlyFans-master/helpers/main_helper.py", line 939, in process_names
    result = await module.start_datascraper(authed, username, site_name)
  File "/mnt/d/OnlyFans-master/modules/onlyfans.py", line 161, in start_datascraper
    await prepare_scraper(authed, site_name, item)
  File "/mnt/d/OnlyFans-master/modules/onlyfans.py", line 820, in prepare_scraper
    unrefined_set = await tqdm.gather(tasks, **settings)
AttributeError: type object 'tqdm_asyncio' has no attribute 'gather'
```

**Comment by LordAbortion** [comment]
+1

**Comment by LordAbortion** [comment]
@bogdanstoian1988 run pip install --upgrade --user -r requirements.txt while the cli is inside the directory containing aforementioned requirements.txt, should clear up that issue. Now it's just not downloading anything but headers for me

**Comment by americanseeder1865** [comment]
Are you using Docker?

**Comment by chenks** [comment]
it appears that it's the verison of tqdm causing the issue.
fix is to manually downgrade tqdm back to 4.59.0
